template/setup.cfg

```diff
@@ -0,0 +1,34 @@
+[metadata]
+name = project_xyz
+version = 0.1.0
+description = A short description of the project
+long_description = file: README.md
+long_description_content_type = text/markdown
+author = Your Name
+author_email = your.email@example.com
+url = https://github.com/yourusername/project_xyz
+classifiers =
+    Programming Language :: Python :: 3
+    License :: OSI Approved :: MIT License
+    Operating System :: OS Independent
+
+[options]
+package_dir =
+    = src
+packages = find:
+python_requires = >=3.8
+install_requires =
+    # Add your project dependencies here
+
+[options.packages.find]
+where = src
+
+[tool:pytest]
+testpaths = tests
+addopts = --cov=project_xyz --cov-report=term-missing --cov-report=xml
```

Comment:
Is this some pytest specific coverage addon? It would probably be better to use the well-established `coverage` package. See for example https://stackoverflow.com/a/62224494